src/hotspot/share/classfile/bytecodeAssembler.hpp

```diff
@@ -131,12 +130,16 @@ class BytecodeConstantPool : ResourceObj {
   ConstantPool* _orig;
   GrowableArray<BytecodeCPEntry> _entries;
   IndexHash _indices;
+  int _orig_cp_added;
```

Comment:
I think the names `_entries` and `indices` could be improved to be more clear. Specifically, `_entries` should refer to the fact that it is a temporary copy of the constant pool that is used to generate the new constant pool.

Reply:
I renamed these fields to _added_entries and _index_map which makes the code a bit clearer, and added a couple of comments here.